sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeTestUtils.scala

```diff
@@ -39,7 +41,7 @@ object DateTimeTestUtils {
 
   val UTC_OPT = Option("UTC")
 
-  val ALL_TIMEZONES: Seq[TimeZone] = TimeZone.getAvailableIDs.toSeq.map(TimeZone.getTimeZone)
+  val ALL_TIMEZONES: Seq[ZoneId] = ZoneId.getAvailableZoneIds.asScala.map(getZoneId).toSeq
```

Comment:
I'm wondering if we should only test the `outstandingZoneIds` instead of `ALL_TIMEZONES`. The test can be very slow if we test all timezones.

Reply:
We already discussed this some time ago, see https://github.com/apache/spark/pull/22657 . We replaced `ALL_TIMEZONES` by outstanding time zones in some places but decided to leave in date-time tests. @srowen proposed to run tests for `ALL_TIMEZONES` in parallel, see https://github.com/apache/spark/pull/22672 (https://github.com/apache/spark/pull/22672#issuecomment-427926845). Total time of testing all time zone in parallel ~0.24 seconds. Not so much, I think.